node/pkg/driver/device_connectivity/device_path_ppc64le.go

```diff
@@ -0,0 +1,4 @@
+package device_connectivity
+
+// for PowerPC, no FC subsystem prefix is needed
```

Comment:
better change to "for Power PC there is no FC subsystem prefix"

Reply:
do you know this for a fact? because I don't

Reply:
This is what you did here. Prefix is empty string -> no prefix.

Reply:
doesn't mean that I know that FC **never** (not even outside CSI context) has a prefix on PowerPC

Reply:
as I understood from the other conversations, I'm removing all 3 comments